src/main/java/org/sagebionetworks/bridge/services/AssessmentResourceService.java

```diff
@@ -218,7 +218,7 @@ public void deleteResourcePermanently(String appId, String assessmentId, String
         checkArgument(isNotBlank(assessmentId));
         
         // Must have published the assessment already before you move resources
-        Assessment assessment = assessmentService.getLatestAssessment(SHARED_APP_ID, assessmentId);
+        Assessment assessment = assessmentService.getLatestAssessment(SHARED_APP_ID, null, assessmentId);
```

Comment:
So there are two ways here to verify the caller owns the assessment, one using the query, one checking the object, do we need both?

Reply:
We do. They complement each other